### ISSUE TITLE
Email delivery fails if author is "Anon <>"

### ISSUE THREAD

**Issue by sudoman** [git-multimail/git-multimail#190]
````
git commit --author="Anon <>"
git push
````

Exim rejected the commit message email with this complaint:

````
rejected after DATA: missing or malformed local part: failing address in "Reply-To:" header is: Anon <>
````

The Exim process on the local server didn't complain before sending it to the next Exim server, but delivery from the second Exim server failed with the above message. This implies that the issue may depend on the configuration of Exim.

**Comment by mhagger** [comment]
@sudoman: What do you think should happen in this scenario?

Is the issue that the whole email was rejected; i.e., that it wasn't sent to other (non-malformed) recipients?

**Comment by sudoman** [comment]
@hhagger Yes, the email should have been successfully delivered to whatever email address(s) were configured for delivery. Otherwise, trackers of the repo may miss that change in the git history. As for what happens to the Reply-To address, maybe that should be excluded from the email if a properly formed address cannot be generated.
